components/post-image/PostImage.tsx

```diff
@@ -18,19 +18,18 @@ interface IProps {
 
 const PostImage = ({ src, alt }: IProps) => {
   const [loaded, setLoaded] = React.useState(false);
-  const imageProportions = 'pb-[calc((8/17)*100%)]';
-  const imageTransition = 'transition-all ease-in delay-500 duration-500';
+  const imageTransition = 'transition-opacity duration-500 delay-500';
 
   const { dispatchModal } = useModal();
   return (
     <div className="relative">
       <div
-        className={`${loaded ? '-z-10 opacity-0' : 'z-10 opacity-100'} absolute top-0 left-0 right-0 bottom-0 ${imageProportions} ${imageTransition}`}
+        className={`${loaded ? '-z-10 opacity-0' : 'z-10 opacity-100'} absolute top-0 left-0 right-0 bottom-0 mumble-image ${imageTransition}`}
```

Comment:
is mumble-image an identifier or a global css class?

Reply:
mumble image is a global class.